assembly-looknfeel-css.xml

```diff
@@ -0,0 +1,18 @@
+<assembly>
+    <id>css</id>
+    <formats>
+        <format>zip</format>
+    </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <fileSets>
+        <fileSet>
+            <directory>${project.build.directory}/css</directory>
+            <outputDirectory></outputDirectory>
+            <filtered>true</filtered>
+        </fileSet>
```

Comment:
I think this fileset can be removed. Since there is no version fille, I think we have nothing to replace.